website/documents/forms.py

```diff
@@ -21,7 +21,6 @@ class DocumentFileInput(widgets.ClearableFileInput):
     def get_context(self, name, value, attrs):
         context = super().get_context(name, value, attrs)
         if hasattr(value, "url"):
-            doc = models.Document.objects.get(Q(file_en=value) | Q(file_nl=value))
```

Comment:
I am not 100% this is something we should do. The Dutch documents should actually stay available to members right?

Reply:
And are Dutch board documents and/or gm documents in an archive somewhere?

Reply:
> Not about this line but the file. I am not 100% this is something we should do. The Dutch documents should actually stay available to members right? Minutes maybe not, but the articles of association and the privacy policy?

I think for the general docs we can make a migration that converts these into "english" documents, with a suffix in the name. The privacy policy and minutes and stuff I'm not sure about yet.

> And are Dutch board documents and/or gm documents in an archive somewhere?

Yes the board has all of them in the Drive

Reply:
I'm not sure what you mean by converting them, and how we are going to show them. The files themselves will never be deleted.